src/test/java/lotto/domain/WinningLottoTest.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class WinningLottoTest {
+
+    WinningLotto winningLotto;
```

Comment:
인스턴스 변수의 접근 제한자는 private으로 구현합니다.

```suggestion
    private WinningLotto winningLotto;
```